fixture.yaml

```diff
@@ -0,0 +1,34 @@
+generic:
+  time_range_1:
+    - !!python/tuple ['2010-01-12 10:00:00', '2010-01-12 12:00:00']
```

Comment:
Nice, I didn't know about Python-specific tags in PyYAML - as you probably realise, this will prevent you from re-using the data in a different programming language... but probably that's also not a good idea in the first place 🤔